ios/examples/run-local-test/specs/local_test.js

```diff
@@ -3,7 +3,7 @@ var assert = require('assert');
 
 describe('BrowserStack Local Testing', () => {
   it('can check tunnel working', async () => {
-    var searchSelector = await $(`~Test BrowserStackLocal connection`);
+    var searchSelector = await $(`~TestBrowserStackLocal`);
```

Comment:
Local sample apps have been recently updated and locator need to be in sync with other sample repos

Reply:
Is `accessibility id` going to change often? If so can we use some other selector.. 
if is a one time change from local side then should be fine

Reply:
All other sample repos already using `TestBrowserStackLocal` and its kind of one time change only